src/common/gui/CHSwitch2.h

```diff
@@ -33,6 +33,7 @@ class CHSwitch2 : public VSTGUI::CHorizontalSwitch, public Surge::UI::SkinConsum
    // This matches the paint offset
    virtual int getIValue() { return (int)(value * (float)(rows * columns - 1) + 0.5f); }
    
+   int mouseDowns;
```

Comment:
You need to initialize mouseDowns to 0 in the CHSwitch constructor (so around line 29 of this file do a mouseDowns = 0)